### ISSUE TITLE
Remove footer page type

### ISSUE THREAD

**Issue by geoo89** [unicef/iogt#1082]
To ensure no footers are in use, we added code that makes it impossible to add new footers (#1070).
Once this is deployed on all sites, we can check that no footer pages have been added (and remove them if necessary) and then remove the footer type from the codebase.

**Comment by cbunicef** [comment]
Should this also change the "Footer Index Page" type?

**Comment by geoo89** [comment]
No, that is unaffected. We only want to remove the footer page type as it is identical to articles. The footer index page can contain different page types, which _serve_ as footers.

**Comment by cbunicef** [comment]
Yes I understand, but wondering if it is confusing to have a "Footer Index Page" type when there is no longer a Footer Page type.

I get now that "Footer Index Page" is necessary. Is there a better name we can use? I don't have an answer, let's keep it as-is for now.

**Comment by cbunicef** [comment]
As discussed "X Index Page" doesn't seem to be a spec in Wagtail. We use it much more than the base software, original Wagtail may only have "Section Index Page". Let's leave Footer Index Page and Miscellaneous Index Page.
